includes/functions-general.php

```diff
@@ -593,7 +593,8 @@ function wpas_tickets_dropdown( $args = array(), $status = '' ) {
 	$options = '';
 
 	foreach ( $tickets as $ticket ) {
-		$options .= "<option value='$ticket->ID'>$ticket->post_title</option>";
+		$is_selected = isset($args['selected']) && $args['selected'] == $ticket->ID ? true : false;
```

Comment:
## Security Issues

1. Use strict comparison (`===` instead of `==`)
2. Use Yoda conditions (`$ticket->ID === $args['selected']`)

## Coding Standards

Please stick to the [WP coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/) regarding spacing.

```
$is_selected = isset( $args['selected'] ) && $args['selected'] == $ticket->ID ? true : false;
$options .= "<option value='$ticket->ID'" . ( $is_selected ? ' selected="selected"' : '' ) . ">$ticket->post_title</option>";
```